cypress/integration/inventory/keyboard-shortcuts.spec.js

```diff
@@ -31,4 +31,23 @@ describe('ui-inventory: keyboard shortcut', () => {
     InventoryMainButton.waitModalLoading('Keyboard shortcuts');
     InventoryMainButton.verifyInventoryDropdownIsShown('false');
   });
+
+  it('C345298 Keyboard Shortcut. Test the functionality of the different shortcut keys', { tags: [testTypes.smoke] }, () => {
+    InventoryMainButton.verifyInventoryDropdownIsShown('false');
+
+    InventoryMainButton.openInventoryMenu();
+    InventoryMainButton.openShortcuts();
+
+    InventoryMainButton.waitModalLoading('Keyboard shortcuts');
+    InventoryMainButton.createNewRecordByShortcuts();
+    InventoryMainButton.closeShortcutsViaUi();
+    InventoryMainButton.fillInstanceInfoViaUi();
+    InventoryMainButton.editInstanceInfoViaUi();
```

Comment:
you can find in code already created fragements related with inventory instance. Try to use it instead of   InventoryMainButton.

Reply:
Done